### DISCUSSION TITLE
Inspecting local variables?

### DISCUSSION THREAD

**Discussion by alfiedotwtf** [SeaQL/FireDBG.for.Rust#52] [Q&A] [answered]
I've been wanting something like this since forever! Thanks for such a cool project 🤩

In the README, one of the features is:

    "Visually inspect local variables, function arguments and its return value"

I can see function arguments and return values, that's super on its own, but I've tried looking everywhere but can't seem to find how to inspect variables?

Thanks!

**Comment by ArturAssisComp** [comment, marked as answer]
It is possible to inspect variables using the firedbg-trace-macro. Take a look at [this section](https://firedbg.sea-ql.org/blog/2023-12-13-getting-started/#firedbg-trace-macro).
